src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double area() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Right triangle color: " + getColor()
+                + ", First leg length = " + firstLeg
+                + ", Second leg length = " + secondLeg
+                + "Area = " + area());
```

Comment:
There is a missing comma and space in the `draw` method output before 'Area = '. It should be ', Area = ' for better readability.